src/launch.py

```diff
@@ -183,10 +183,7 @@ def main():
     argv = [sys.argv[0]]
     argv.extend(extra_args)
     argv.extend(args.remain)
-    try:
-        app = OpenShotApp(argv)
-    except Exception:
-        app.show_errors()
```

Comment:
app was never initialized here... and below a few lines, we call `app.gui()` which displays the errors... if a gui is created.